components/eam/src/chemistry/aerosol/wetdep.F90

```diff
@@ -438,59 +428,38 @@ subroutine rain_mix_ratio(t, pmid, sumppr, ncol, rain)
    real(r8), intent(out) :: rain(pcols,pver) ! mixing ratio of rain [kg/kg]
 
    ! Local variables:
-   integer  i,k
+   integer  icol,kk
 
    ! constants used in fallspeed calculation; taken from findmcnew
    real(r8) :: convfw
    real(r8) :: rho     !air density
    real(r8) :: vfall
 
    ! define the constant convfw. taken from findmcnew, do not find the reference
-   ! of the equation  -- by Shuaiqi when refactoring
+   ! of the equation  -- by Shuaiqi when refactoring for C++
```

Comment:
It's too bad we don't know where this equation and the associated parameters come from. Is it worth trying to back out what the numbers mean with dimensional analysis?

Reply:
found and added a reference